gems/quilt_rails/test/generators/quilt/react_app/react_app_generator_test.rb

```diff
@@ -12,6 +12,13 @@ class QuiltReactAppGeneratorTest < Rails::Generators::TestCase
     prepare_destination
   end
 
+  test "set javascript path" do
+    run_generator
+    assert_file "config/application.rb" do |app_config|
+      assert_match "config.javascript_path = \"ui\"", app_config
```

Comment:
@gmcgibbon the reason why this is failing is because it cannot find `config/application.rb`
what should happen if there if no `config/application.rb`?
unless that is just a problem of my test case?